### DISCUSSION TITLE
XDM Youtube Playlist download

### DISCUSSION THREAD

**Discussion by nb5691** [subhra74/xdm#575] [General]
Hi, I have XDM Version 7.2.11 with Java Eclipse OpenJ9 11.0.6 on Linux/Ubuntu 20.04 LTS desktop. I am unable to download youtube playlist. When I paste the youtube playlist link and press "Find", only blank screen appears. I also tried giving username and password in advanced settings (youtube gmail credentials). same result. Please help.

**Comment by subhra74** [comment]
This is a bug, will be fixed in next version

**Reply by moedwyn** [reply]
reserved
thanks subhra,,,, 👍👍👍

**Reply by BraINstinct0** [reply]
any update on this?

**Reply by R1sh1Par1kh** [reply]
thanks

**Reply by delsonKashabano** [reply]
still 2023 I DONT SEE THE SOLUTION TO DOWNLOAD YOUTUBE PLAYLISTS

**Comment by nb5691** [comment]
Thank you sir.

**Reply by karume-lab** [reply]
I need more bullets.

**Comment by subhra74** [comment]
The fix will be available in October

**Reply by vehosah** [reply]
at last a new update

**Reply by Vuzereus** [reply]
It is weird, I think the URL filter somehow is broken.
https://www.youtube.com/playlist?list=[playlist id] works, but the selection dialog box shows all video as "watch".
Also, when the video played and switch to next video only then the Monitor triggered.
